configs/peer/config.json

```diff
@@ -75,7 +75,7 @@
       "max": 128
     },
     "WASM_RUNTIME_CONFIG": {
-      "FUEL_LIMIT": 20000000,
+      "FUEL_LIMIT": 25000000,
```

Comment:
I think this limit shouldn't be applied to the monolithic runtime validator

Reply:
I tried `23_000_000` and it works, will replace it.

This is generally needed for `mint_nft_for_every_user_every_1_sec()` test, where monolithic validator checks WASM smartcontract

Reply:
what I'm saying is that we might want a different parameter for limiting triggers vs limiting monolithic validator. The 2nd one should have a much larger limit for execution time

Reply:
How would we differentiate between them, if we execute validator on every instruction execution from trigger/smart contract?

Reply:
> I think this limit shouldn't be applied to the monolithic runtime validator

I'm not quite agree with this, failing with fuel limit error might be good starting point for the users/validator devs to investigate what is going on with iroha. If validator will be limitless iroha will just freeze in case of errors in validator.

Reply:
> I think this limit shouldn't be applied to the monolithic runtime validator

Imo failing with fuel limit error might be good starting point for the users/validator devs to investigate what is going on with iroha. If validator will be limitless iroha will just freeze in case of errors in validator.

Reply:
> > I think this limit shouldn't be applied to the monolithic runtime validator
> 
> Imo failing with fuel limit error might be good starting point for the users/validator devs to investigate what is going on with iroha. If validator will be limitless iroha will just freeze in case of errors in validator.

I'm not saying it should be limitless, that would be insane. I'm saying that we should consider having a different configuration parameter for monolithic validator fuel limit vs smart contract fuel limit. I'm saying this because I expect a big difference in the actual value of these two

Reply:
Aha, sorry for misunderstanding. Yeah, different limit might be good idea, because runtime validator will probably take more time than ordinary trigger, but we can expect mote then one trigger. And currently we are increasing limit for all of them which can add up.

Reply:
@mversic , created #3354 . If you think we're done here, close the issue, please